lib/src/async_environment.dart

```diff
@@ -837,7 +809,7 @@ class AsyncEnvironment {
   Module toModule(CssStylesheet css, ExtensionStore extensionStore) {
     assert(atRoot);
     return _EnvironmentModule(this, css, extensionStore,
-        forwarded: _forwardedModules);
+        forwarded: _forwardedModules?.keys.toSet());
```

Comment:
It would probably be more efficient to use a `MapKeySet` here, unless there's a reason it *shouldn't* update if `_forwardedModules` is modified.